.github/workflows/community-pr.yml

```diff
@@ -0,0 +1,42 @@
+name: Label community PRs
+on:
+  pull_request_target:
+    types: [opened]
+permissions:
+  issues: write
+  pull-requests: write
+jobs:
+  add_milestone:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'dotnet/roslyn-analyzers' && github.event.pull_request.author_association != 'MEMBER' && github.event.pull_request.author_association != 'OWNER' }}
+    steps:
+    - name: Get label data
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        ORGANIZATION: dotnet
```

Comment:
I think the repo and org can come from the `github` context if you don't want to hardcode it. 
e.g. ${{ github.repository }}
https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Reply:
> Or can this be triggered by forks

I kept running into problems with microsoft/vs-extension-testing where actions ran in forks even though the documentation says they don't.